tools/issue-labeler/main.go

```diff
@@ -1,23 +1,40 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"os"
+	"sort"
+	"strings"
 
 	"github.com/golang/glog"
 	"gopkg.in/yaml.v2"
 )
 
+var flagBackfill = flag.String("backfill-date", "", "run in backfill mode to apply labels to issues filed after given date")
```

Comment:
Nit: Could make this an explicit `flagBackfillDate` or similar, so that it isn't confused with being a boolean flag.

Reply:
Done